### ISSUE TITLE
App Running, calls to API working, MQTT posting labels but no data coming through.

### ISSUE THREAD

**Issue by drkpxl** [wubbl0rz/FiatChamp#71]
App install was easy. Adding config variables was easy, config logs shows successful connection and I get back stats labels but no stats. I waited a few hours and haven't seen anything other then the labels. I am quite experienced with Home Assistant and MQTT but this seems to be an API issue for Jeep's in America?

**Comment by Jster42** [comment]
```
[08:29:28 DBG] {
  "Userid": "SXMAUSEN_xxxxxxxxxx",
  "Version": 0,
  "Vehicles": []
}
```
This is what I see in the logs after successful api connections for Jeep, I am assuming that the api is not returning the associated vehicle,  are you seeing the same in debug?

**Comment by drkpxl** [comment]
No, I actually get back my nickname for it and all the sensors with empty values. Not null just empty

**Comment by tarfan22** [comment]
No, I actually get back my nickname for it and all the sensors with empty values. Not null just empty same here with Jeep
